CRM/Contact/Page/View/UserDashBoard.php

```diff
@@ -186,9 +181,9 @@ public function buildUserDashBoard() {
     $this->assign('dashboardElements', $dashboardElements);
 
     // return true when 'Invoices / Credit Notes' checkbox is checked
-    $this->assign('invoices', $this->_userOptions['Invoices / Credit Notes']);
+    $this->assign('invoices', $dashboardOptions['Invoices / Credit Notes']);
 
-    if (!empty($this->_userOptions['Groups'])) {
+    if (!empty($dashboardOptions['Groups'])) {
```

Comment:
This variable is only used within this function. So ok with this if it helps set the scene for further refactoring